scripts/deploy_to_test_pypi_org.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
Please revert this change. Deploying to test repo can be useful for some testing.